player/mcts/mcts_node.py

```diff
@@ -48,7 +48,7 @@ def sample_mean(self) -> float:
 
     @property
     def eval(self) -> float:
-        return self.sample_mean + np.sqrt(np.log(self.parent.number_of_runs) / self.number_of_runs)
+        return self.sample_mean + np.sqrt(2 * np.log(self.parent.number_of_runs) / self.number_of_runs)
```

Comment:
I changed the formula so that this value of 2 is `c` parameter which is property of node:
![image](https://github.com/Michalski-Artur/MCTS-Connect-4/assets/79529984/2308ce4d-414a-4809-b896-60a53c27acb8)

I think it might be more flexible solution

Reply:
but 2 is the constant in the formula, not c parameter